contrib/update-storage-times.py

```diff
@@ -109,11 +106,18 @@ def fix_metric(metric):
         res = 0
     else:
         LOG.debug('Retention will be %s' % retention)
+        # record file owner/group and perms to set properly after whisper-resize.py is complete
+        perms = os.stat(metric).st_mode
+        owner = os.stat(metric).st_uid
+        group = os.stat(metric).st_gid
```

Comment:
Better to run stat() just once and save the result in a tuple, that saves two system calls, ex:
st = os.stat(metric)
...
os.chmod(metric, st.st_mode)
os.chown(metric, st.st_uid, st.st_gid)